### ISSUE TITLE
iOS Privacy Manifest 

### ISSUE THREAD

**Issue by Presheda** [crazecoder/open_file#299]
Is this package handling iOS privacy manifest ? 

I got the ios privacy manifest warning after  adding this package and pushing a build. 

`TMS-91108: Invalid privacy manifest - The PrivacyInfo.xcprivacy file from the following path is invalid: “Frameworks/Reachability.framework/ReachabilitySwift.bundle/PrivacyInfo.xcprivacy”.`

<img width="746" alt="image" src="https://github.com/user-attachments/assets/63dd2774-970e-4153-a6a1-40bfe0387b2d">

**Comment by crazecoder** [comment]
On iOS, it doesn't ask for any privacy-related permissions, so maybe you should check that out

**Comment by Presheda** [comment]
Sorry for the troubles @crazecoder , it was something else

**Comment by Presheda** [comment]
.

**Comment by Utkarsh-Mirrorar** [comment]
HI @Presheda , how did you fix the issue?

**Comment by Presheda** [comment]
@Utkarsh-Mirrorar 

There's most likely a package which is referencing Reachability in your project

I was able to resolve mine, in my case it was the connectivity_plus package that used the Reachability, I believe i was using and older version of the package too. Another plus is that i was no longer using the connectivity_plus package so i just removed it.

I submitted a build and appstore didn't send the warning anymore.

Search for Reachability in your podfile.lock and see which package is referencing it. Ensure the package are upto date.

**Comment by praveenraman** [comment]
Yes! I also faced this problem. I searched a lot but did not find any solution. Then I searched a bit on YouTube and Google and found the solution.
I Share you step by step solution.
1. Frist you create archive file.
2. Then Right click Archive file. and select  "Generate privacy report " save you laptop.
3. Then open download file and see missing file in bottom.
4. Then search you missing privacy you project folder (PrivacyInfo.xcprivacy) like ReachabilitySwift
5. Then Open Privacy Source code and copy past other success privacy and edit acording.
<img width="293" alt="Screenshot 2024-11-27 at 10 16 22 AM" src="https://github.com/user-attachments/assets/958d10a7-a4cc-4c72-8570-905457a6c5e9">
Like this
<plist version="1.0">
<dict>
	<key>NSPrivacyTracking</key>
	<false/>
	<key>NSPrivacyTrackingDomains</key>
	<array/>
	<key>NSPrivacyCollectedDataTypes</key>
	<array/>
	<key>NSPrivacyAccessedAPITypes</key>
	<array>
		<dict>
			<key>NSPrivacyAccessedAPIType</key>
			<string>NSPrivacyAccessedAPICategoryUserDefaults</string>
			<key>NSPrivacyAccessedAPITypeReasons</key>
			<array>
				<string>CA92.1</string>
			</array>
		</dict>
	</array>
</dict>
</plist>
